functional/platform/nspawn.go

```diff
@@ -407,12 +407,11 @@ UseDNS no
 			return
 		default:
 		}
-		// TODO(jonboulle): probably a cleaner way to check here
-		if _, _, e := nc.nsenter(nm.pid, "systemd-analyze"); e == nil {
+		// NOTE(bcwaldon): check something that requires both systemd and dbus to be happy
```

Comment:
whoops - shouldn't have made this change. I had changed this to run a systemctl command, but forgot to revert the comment back. Will remove